### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ReamFOX**

### CODE REVIEW

File: src/main/java/com/farion/onlinebookstore/OnlineBookStoreApplication.java [hunk 1]
```diff
@@ -1,41 +1,16 @@
...
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class OnlineBookStoreApplication {
 	@Autowired
 	private BookService bookService;
```
Comment (SkillfulDev) [inline src/main/java/com/farion/onlinebookstore/OnlineBookStoreApplication.java]: Why do you need it?

**Review by SkillfulDev — CHANGES_REQUESTED** [review]
The code looks good for this task. But if you take a look at the first task about creating repo , you will see that you didn`t add CI at all. So your code doesn`t check by CI.

File: src/main/java/com/farion/onlinebookstore/OnlineBookStoreApplication.java (hunk 1 above)
Comment (ReamFOX) [reply src/main/java/com/farion/onlinebookstore/OnlineBookStoreApplication.java]: Oh, I forgot to delete it, thanks

File: checkstyle.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0"?>
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (andrii-hoienko) [inline checkstyle.xml:1]: What is the reason to change it?

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
Left minor comment

**Pull request closed without merging**